libc/tzcode/localtime.c

```diff
@@ -274,7 +274,7 @@ settzname(void)
     tzname[1] = wildabbr;
 #ifdef USG_COMPAT
     daylight = 0;
-    timezone = 0;
+    timezone = -sp->ttis[sp->defaulttype].tt_gmtoff;
```

Comment:
|timezone| stores the negative offset from GMT? The rest of the code agrees, but it still makes me wonder.